src/SambaVideos/AbstractResource.php

```diff
@@ -99,6 +99,32 @@ public function search(array $params = [])
         }
     }
 
+    /**
+     * @param array $params
+     *
+     * @return array
+     * @throws RequestException
+     */
+    public function searchHeaders(array $params = [])
```

Comment:
@TheMFhalls Não faz sentido fazer uma requisição apenas para recuperar os _headers_. Você já tem isso pronto no object `$this->response`. Basta criar um método para retornar esse objeto ou então os _headers_ através dele.

```suggestion
    public function getResponse() {
        return $this->response;
    }
```

```suggestion
    public function getResponseHeaders() {
        return $this->response->headers;
    }
```